pkg/plugins/policies/donothingpolicy/api/v1alpha1/donothingpolicy.proto

```diff
@@ -35,23 +35,23 @@ message DoNothingPolicy {
   message To {
     // TargetRef is a reference to the resource that represents a group of
     // destinations.
-    kuma.common.v1alpha1.TargetRef targetRef = 1;
+    kuma.common.v1alpha1.TargetRef targetRef = 1 [ (doc.required) = true ];
 
     // Default is a configuration specific to the group of destinations
     // referenced in 'targetRef'
-    Conf default = 2;
+    Conf default = 2 [ (doc.required) = true ];
   }
 
   repeated To to = 2;
 
   message From {
     // TargetRef is a reference to the resource that represents a group of
     // clients.
-    kuma.common.v1alpha1.TargetRef targetRef = 1;
+    kuma.common.v1alpha1.TargetRef targetRef = 1 [ (doc.required) = true ];
```

Comment:
I can't tell from the PR, what are the effects of adding this?

Reply:
It's just an extra field in the generated docs. But docs for policies are not generated yet (reminds me we need to add this :) )